apps/src/templates/studioHomepages/CourseBlocksStudentGradeBands.story.jsx

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
 import CourseBlocksStudentGradeBands from './CourseBlocksStudentGradeBands';
+import Responsive from '../../responsive';
+
+const responsive = new Responsive({
```

Comment:
At the very least let's pull this out and make it a exported constant in Responsive.js - you use it about 5 times in this PR.